packages/api/src/@types/custom.d.ts

```diff
@@ -1,7 +1,10 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 namespace Express {
   interface Request {
+    id: string | undefined;
     cxId: string | undefined;
+    patient: any | undefined; //eslint-disable-line @typescript-eslint/no-explicit-any
+    facility: any | undefined; //eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Couldn't use the actual type otherwise this wasn't being recognized as additional type definition. It doesn't impact the reading, though, and the code that uses those is defined once (so not a lot of maintenance that we really need types).